app/javascript/components/visualization/DotPlot.jsx

```diff
@@ -353,3 +355,16 @@ export function logMorpheusPerfTime(target, plotType, genes) {
 
   log(`plot:${plotType}`, { perfTime, genes })
 }
+
+/** Log performance of loading JSON datasets for Morpheus */
+export function logFetchMorpheusDataset(perfTimes, cluster, annotation, genes) {
+  const props = {
+    perfTimes,
+    cluster,
+    annotName: annotation.name,
+    annotType: annotation.type,
+    annotScope: annotation.scope,
+    genes
+  }
+  log(`dot-plot:dataset`, props)
```

Comment:
This concerned me, because it doesn't conform to the performance logging patterns used for scatter plots or violin plots.  However, performance logging for Morpheus is already quite idiomatic, so that's a pre-existing issue.  It also seems not exactly trivial to bring into conformance, which makes refining all that out-of-scope.

Reply:
Yeah, I looked at what we were logging for Morpheus and it was definitely not in line with the Plotly-based metrics.  I tried to make at least the data reported to be consistent.